src/main/java/org/testobject/rest/api/model/StartXcuiTestResponse.java

```diff
@@ -0,0 +1,28 @@
+package org.testobject.rest.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StartXcuiTestResponse {
+
+	private final long testReportId;
+	private final String testReportUrl;
+
+	public StartXcuiTestResponse(
+			@JsonProperty("testReportId") long testReportId,
+			@JsonProperty("testReportUrl") String testReportUrl
```

Comment:
In `product` the resource it as `testReportURL` so this will be `null` here. Should it be changed in product or here?

Reply:
i will change it here